apps/user-ms/src/main/java/it/pagopa/selfcare/user/controller/UserController.java

```diff
@@ -138,6 +139,7 @@ public Uni<List<UserInstitutionResponse>> findAllByIds(@QueryParam(value = "user
 
     @Operation(summary = "Retrieve all users according to optional params in input")
```

Comment:
It is not clear what is the scope of this API...."Retrieve all users" but the path is notification, you could add more comments to explain the behaviour